src/components/Dialog/DialogSection.react.js

```diff
@@ -79,7 +86,7 @@ const DialogSection = props => {
     return DialogData.noComponent;
   };
 
-  const { size, Component, fullScreen = false } = getDialog();
+  const { size, Component, fullScreen = false, padding } = getDialog();
```

Comment:
Add a default value of padding incase no padding is passed.

Reply:
@akshatnitd If no padding is passed is being handled in shared Container. The reason I have not passed a default prop is because default value will change according to the screen width. 
what i can do is `padding = isMobileView() ? '' : ''`. Shall I give a default value like this ?